src/ontodia/widgets/toolbar.tsx

```diff
@@ -41,7 +41,7 @@ export class EditorToolbar extends React.Component<Props, void> {
 
         return (
             <button type='button' className='saveDiagramButton ontodia-btn ontodia-btn-primary'
-                    onClick={this.props.onSaveDiagram}>
+                onClick={this.props.onSaveDiagram}>
```

Comment:
Please discard any unnecessary formatting changes in this and `elementLayer` files. These changes make it harder to review and to investigate code history in the future.

Reply:
done